dataprofiler/profilers/graph_profiler.py

```diff
@@ -401,7 +402,17 @@ def _get_continuous_distribution(
 
                 for distribution in distribution_candidates:
                     # compute fit, mle, kolmogorov-smirnov test to test fit, and pdf
-                    fit = distribution.fit(df)
+
+                    # scipy 1.11.0 updated the way they handle
+                    # the loc parameter in fit() for lognorm
+                    scipy_version = int(
+                        importlib.metadata.version("scipy").replace(".", "")
+                    )
```

Comment:
My concern is the int \ float conversion of the version. This would error if they did a post fix etc. We might want to consider for graph installing packaging so we can do a comparison between versions unless there's an easier way without having to install another repo.

If we need to, we can choose to install it only on the graph since that's where this is being used.